CITATION.cff

```diff
@@ -0,0 +1,23 @@
+# Parser settings.
+cff-version: 1.2.0
+message: Please cite this crate using these information.
+
+# Version information.
+date-released: 2022-11-12
+version: 0.4.23
+
+# Project information.
+abstract: Date and time library for Rust
+authors:
+  - alias: .
```

Comment:
This is an optional field to put the person's pseudonym.  I recommend to use it for the GitHub user name as the repository exposes its contributors' user names anyway.  I entered `.` as a placeholder -- and to make this configuration pass the validation.

The CFF team maintains a [CLI](https://github.com/citation-file-format/cff-converter-python) for validating CITATION.cffs and also provide a Dockerised image of it from which they built a [validation Action](https://github.com/citation-file-format/cffconvert-github-action).  If you are interested, I can also add a GitHub Action workflow for automatic validation of this configuration file (including the README badge, at option).

Reply:
Yes, would be good to get the file validated on an ongoing basis.

Reply:
Done.  The job usually succeeds within at most 15 seconds.  Including the setup, it should never require more than 30 seconds.